pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
```

Comment:
I used the date returned from `nix-prefetch-git`:

```shell
❯ nix-prefetch-git https://github.com/stark/siji
Initialized empty Git repository in /tmp/git-checkout-tmp-UuH7HSt2/siji/.git/
remote: Counting objects: 11, done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 11 (delta 0), reused 6 (delta 0), pack-reused 0
Unpacking objects: 100% (11/11), done.
From https://github.com/stark/siji
 * branch            HEAD       -> FETCH_HEAD
Switched to a new branch 'fetchgit'
removing `.git'...

git revision is 95369afac3e661cb6d3329ade5219992c88688c1
path is /nix/store/qv0p80c5pg53qpsvdjbnzk4sdgqf8gdi-siji
git human-readable version is -- none --
Commit date is 2016-05-14 00:30:21 +0530
hash is 1408g4nxwdd682vjqpmgv0cp0bfnzzzwls62cjs9zrds16xa9dpf
{
  "url": "https://github.com/stark/siji",
  "rev": "95369afac3e661cb6d3329ade5219992c88688c1",
  "date": "2016-05-14T00:30:21+05:30",
  "sha256": "1408g4nxwdd682vjqpmgv0cp0bfnzzzwls62cjs9zrds16xa9dpf",
  "fetchSubmodules": true
}
```

But you're right that the commit is from the 13th... Any idea why?